.github/workflows/build.yml

```
@@ -0,0 +1,165 @@
+        run: |
+          $buildVersion = "3.3.$([int]$env:BUILD_VERSION_OFFSET + ${{ github.run_number }}).0"
+          echo "BUILD_VERSION=$buildVersion" >> $env:GITHUB_ENV
+          echo "build-version=$buildVersion" >> $env:GITHUB_OUTPUT
+          Write-Output "Building version $buildVersion"
+
+      - name: 'Bootstrap vcpkg'
+        shell: cmd
+        run: .\XInputBridge\vcpkg\bootstrap-vcpkg.bat -disableMetrics
+
+      - name: 'Configure vcpkg NuGet binary cache'
+        if: ${{ secrets.VCPKG_NUGET_TOKEN != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)